src/sql/media/icons/backup-toolbar.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Did you reach out to get dark-mode icons as well?

Reply:
Yup. Raden will get back to me after checking with visual designers.